vpr/src/route/rr_graph_uxsdcxx_serializer.h

```diff
@@ -16,7 +16,7 @@
 #include "check_rr_graph.h"
 #include "rr_graph2.h"
 #include "rr_graph_indexed_data.h"
-
+#include "globals.h"
```

Comment:
Remove the ``global.h``, we should minimize the use of global variables.

Reply:
After removing global variables and above changes does I need to run Qor tests again?

Reply:
Yes.

Reply:
Oh OK!  But they take much time.

Reply:
No worries. Next time, if you see the CI is green, just request for a code review before running the QoR tests. After we agree on the changes, we run the QoR check. The current QoR looks o.k. now. We will discuss about this PR this Thursday. If you have any update on the QoR, please let me know.

Reply:
In morning I will make changes push if it. If CI is green then I should run QoR tests?

Reply:
Yes.

Reply:
OK thanks.